app/metrics.yaml

```diff
@@ -83,3 +83,52 @@ searches:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-05-01"
+
+tabs:
+  opened:
+    type: event
+    description: >
+      Event for when a tab is opened
+    send_in_pings:
+      - baseline
+    extra_keys:
+      source:
```

Comment:
I get confused. It doesn't match any question at #1609. Is it trying to solve the request of `Number of users who open a new tab, average number of tabs opened/session`?

Reply:
Don't you want to count how many times per session? If so, you might can consider `labeled_counter`.

Reply:
I've migrated this to a `labeled_counter` makes more sense.